test/unit/helpers/ConfigHelper.ts

```diff
@@ -211,8 +213,15 @@ test(`promptOptions 8 - autoRegister = true backwards compatibility for custom i
       autoPrompt: true,
     },
     slidedown: {
-      enabled: true,
-      autoPrompt: false,
+      prompts: [{
```

Comment:
Should keep the old format so we are still testing what could be in the wild as is.

Can use `getPromptOptionsFromFinalAppConfig` to make the tests pass.